html/webappapis/structured-clone/structured-clone.any.js

```diff
@@ -4,6 +4,10 @@
 // META: script=/html/webappapis/structured-clone/structured-clone-battery-of-tests-harness.js
 
 runStructuredCloneBatteryOfTests({
-  structuredClone: (obj, transfer) => self.structuredClone(obj, { transfer }),
+  structuredClone: (obj, transfer) => {
+    return Promise.resolve().then(
+      () => self.structuredClone(obj, { transfer })
+    );
```

Comment:
This would be a tick faster unless I'm mistaken (untested):
```suggestion
    return new Promise(resolve => {
      resolve(self.structuredClone(obj, { transfer }));
    });
```

Reply:
The reason I opened this PR was because Firefox's new `self.structuredClone()` implementation was failing those tests, because it was throwing an exception rather than rejecting a promise. You would have to use a try-catch block to catch the exception, and I'm not sure about the performance of that.

Reply:
An exception in this function is caught and results in a rejected promise.

Reply:
Oh, wait. Exceptions thrown inside the `new Promise` callback do result in a rejected promise. 🤦